bzt/modules/blazemeter.py

```diff
@@ -77,6 +77,7 @@ def prepare(self):
         self.client.user_id = self.parameters.get("user-id", None)
         self.client.data_signature = self.parameters.get("signature", None)
         self.client.kpi_target = self.parameters.get("kpi-target", self.client.kpi_target)
+        self.client.delete_files_before_test = False
```

Comment:
I would say that client class should have default of False and Cloud prov should enable deleting.